index.js

```diff
@@ -35,15 +35,69 @@ var BabelClient = function (config) {
 };
 
 /**
+ * Make a HEAD request to check for new annotations
+ * @param {string} target
+ * @param {string} token
+ * @param {object} params
+ * @param {string} params.delta_token Using a delta_token tells you if there have been new annotations since another one
+ * @callback callback
+ */
+BabelClient.prototype.headTargetFeed = function(target, token, params, callback){
+
+    if(!target){
+        throw new Error('Missing target');
+    }
+    if(!token){
+        throw new Error('Missing Persona token');
+    }
+
+    // params are optional
+    if(typeof params === "function"){
+        callback = params;
+        params = {};
+    }
+
+    // Build up a query string
+    var queryString = '',
+        queryStringParams = [];
+
+    if(!_.isEmpty(params)){
+        for(var i in params){
+            queryStringParams.push(encodeURIComponent(i)+'='+encodeURIComponent(params[i]));
```

Comment:
doesn't the `for(var i i params)` give a list of properties back - not sure what the hasOwnProperty will give me?